### PR TITLE
add and remove tags tech note

### PR DESCRIPTION
**Pull request by prat98**

### CODE REVIEW

**Comment by zkayyali812** [comment]
@prat98 I think the tech note around backup and restore here can be removed as well.
Please confirm and remove in this PR or another PR if so.

**Comment by prat98** [comment]
can confirm that is not an issue any more so it can be removed from the tech note section

**Comment by chrisahl** [comment]
Where are these showing in the Doc?  I don't see a Tech Notes section

**Comment by prat98** [comment]
It's under the technical notes section of the doc section 12. I see it when I generate the doc locally

**Comment by chrisahl** [comment]
Thanks. I now see it way down in AWS...but not in GCP yet.

**Review by chrisahl — APPROVED** [review]

File: stories/topics/tech-note-aws-add-remove-tags.adoc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[id="tech-note-aws-add-remove-tags"]
+
+= Add and Remove Tags
+
+{AAPonAWS} supports stack-level tagging during stack deployment time. A stack-level tag is applied to the stack itself and all tag supported components that make up the stack.
```
Comment (ianf77) [inline stories/topics/tech-note-aws-add-remove-tags.adoc:5]: Perhaps you might say what a tag is?

**Review by ianf77 — COMMENTED** [review]
one comment

File: stories/topics/tech-note-aws-add-remove-tags.adoc (hunk 1 above)
Comment (chrisahl) [reply stories/topics/tech-note-aws-add-remove-tags.adoc:5]: I think AWS users would be familiar with tags, but if you do need to add some text about it, here is one example AWS has:
image

Comment (ianf77) [reply stories/topics/tech-note-aws-add-remove-tags.adoc:5]: Tags are custom fields that you assign to an AWS resource to help organize and identify your instances. {AAPonAWS} supports stack-level tagging during stack deployment time. A stack-level tag, when applied to the stack itself, is also applied to all tag supported components that make up the stack.

File: stories/topics/tech-note-aws-add-remove-tags.adoc [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+Using the add or remove tags playbooks will only update the tags for the individual components of stack and not the tag applied to the stack itself. The stack-level tags will never be modified after deploying {AAPonAWS}.
```
Comment (ianf77) [inline stories/topics/tech-note-aws-add-remove-tags.adoc:7]: playbooks only update the tags for...
Stack-level tags are never modified after deployment.

**Review by ianf77 — COMMENTED** [review]
Some comments

File: stories/assembly-aws-tech-notes.adoc [hunk 3]
```diff
@@ -8,3 +8,4 @@ ifdef::context[:parent-context: {context}]
 {AAPonAWS} is a self-managed deployment. The following are technical notes regarding the {AAPonAWS} deployment.
 
 include::topics/tech-note-aws-add-remove-tags.adoc[leveloffset=+1]
+include::topics/tech-note-ansible-on-clouds-ops-container.adoc[leveloffset=+1]
```
Comment (chrisahl) [inline stories/assembly-aws-tech-notes.adoc:11]: Do you need aws in file name:  tech-note-aws-ansible-on-clouds-ops-container.adoc to follow convention?

Comment (zkayyali812) [reply stories/assembly-aws-tech-notes.adoc:11]: Let us add the `aws` in there to maintain our naming convention like @chrisahl notes @prat98

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**